eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -42,11 +42,11 @@
  */
 public class EidpLoginModule implements LoginModule {
 
-  private Subject subject;
+//  private Subject subject;
   private CallbackHandler callbackHandler;
-  private Map<String, ?> sharedState;
-  private Map<String, ?> options;
-  private boolean debug = false;
+//  private Map<String, ?> sharedState;
+//  private Map<String, ?> options;
+//  private boolean debug = false;
```

Comment:
\[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 2, indentation should be the same level as line 51\.